MediaBrowser.Model/Entities/MetadataProvider.cs

```diff
@@ -14,38 +12,78 @@ public enum MetadataProvider
         Custom = 0,
 
         /// <summary>
-        /// The imdb.
+        /// The IMDb id.
```

Comment:
I think these should all be changed to "{} provider" as they aren't ids.

Reply:
They are our internal provider ids though. but I agree, at least `provider` should be added to the description.